privaterelay/views.py

```diff
@@ -127,8 +129,21 @@ def metrics_event(request: HttpRequest) -> JsonResponse:
 
 @csrf_exempt
 def fxa_rp_events(request: HttpRequest) -> HttpResponse:
+    """MPP-4460: Track more data for FxA relying party exceptions"""
+    with sentry_sdk.new_scope() as scope:
```

Comment:
praise: TIL `new_scope` and [Sentry scopes](https://docs.sentry.io/platforms/python/enriching-events/scopes/).